packages/profile-sync-controller/src/sdk/authentication.ts

```diff
@@ -50,6 +58,29 @@ export class JwtBearerAuth implements SIWEInterface, SRPInterface {
     return await this.#sdk.signMessage(message);
   }
 
+  async pairIdentifiers(
+    pairing: Pair[],
+    signMessage: (message: string) => Promise<string>,
```

Comment:
Eeehm, we actually want the signing to be done by that specific signer. So we can't have 1 signMessage function.

For example we can't have Signer for AddressA to sign AddressB, C, D, etc.

Proposal: We add the signMessage to each "Pair".